pombola/core/models.py

```diff
@@ -573,6 +573,17 @@ def has_ever_been_member_of_ncop_or_national_assembly(self):
             self.has_ever_been_member_of_ncop
         )
 
+    @property
+    def has_ever_been_member_of_prov_legislature(self):
+        return self.position_set.position_titles().exists()
```

Comment:
is this saying "if the position has a title, then it's a provincial legislature"?

but there are all kinds of positions. See how NCOp and National Assembly are implemented - we need to restrict to the relevant slugs https://github.com/OpenUpSA/pombola/pull/107/files#diff-628ab15948cdaab8af240aa616938710a48c4b7f9fb4320307c853372aeb0799R572

https://github.com/OpenUpSA/pombola/pull/107/files#diff-628ab15948cdaab8af240aa616938710a48c4b7f9fb4320307c853372aeb0799R1486